cl/citations/match_citations.py

```diff
@@ -83,26 +92,23 @@ def case_name_query(conn, params, citation, citing_doc):
     return results
 
 
-def get_years_from_reporter(citation):
+def get_years_from_reporter(citation: CitationBase) -> Tuple[int, int]:
```

Comment:
FWIW, I just learned you can just do `-> (int, int)`, instead of the `Tuple` business.

Reply:
Is that preferable?

Reply:
Actually, I was wrong. Nevermind.